tests/e2e/tidbcluster/serial.go

```diff
@@ -985,6 +994,308 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
+			if !ok {
+				klog.Info("tac has no tikv.tidb.pingcap.com/ready-to-scale-timestamp annotation")
+				framework.ExpectNoError(errors.NotFound(""), "tac has no tikv.tidb.pingcap.com/ready-to-scale-timestamp annotation error")
+			}
+			readyToScaleTimestamp, err := strconv.ParseInt(t, 10, 64)
+			if err != nil {
+				framework.ExpectNoError(err, "tikv.tidb.pingcap.com/ready-to-scale-timestamp parse int error")
+			}
+			// check set proper timestamp label
+			framework.ExpectEqual(int(time.Now().Sub(time.Unix(readyToScaleTimestamp, 0)).Seconds()) > 0, true)
+			// check replica should not update to target number
+			framework.ExpectNotEqual(tc.Spec.TiKV.Replicas, int32(4))
+			// check set proper phrase
+			framework.ExpectEqual(tac.Status.TiKV.Phase, v1alpha1.ReadyToScaleOutAutoScalerPhase)
+
+			var firstScaleTimestamp int64
+			err = wait.Poll(10*time.Second, 10*time.Minute, func() (done bool, err error) {
+				tc, err := cli.PingcapV1alpha1().TidbClusters(tc.Namespace).Get(tc.Name, metav1.GetOptions{})
+				if err != nil {
+					return false, nil
+				}
+				// check replicas
+				if tc.Spec.TiKV.Replicas != int32(4) {
+					klog.Infof("tikv haven't auto-scale to 4 replicas")
+					return false, nil
+				}
+				if len(tc.Status.TiKV.Stores) != 4 {
+					klog.Infof("tikv's stores haven't auto-scale to 4")
+					return false, nil
+				}
+				// check annotations
+				if tc.Annotations == nil || len(tc.Annotations) < 1 {
+					klog.Infof("tc haven't marked any annotation")
+					return false, nil
+				}
+				tac, err = cli.PingcapV1alpha1().TidbClusterAutoScalers(ns).Get(tac.Name, metav1.GetOptions{})
+				if err != nil {
+					return false, nil
+				}
+				if tac.Annotations == nil || len(tac.Annotations) < 1 {
+					klog.Infof("tac haven't marked any annotation")
+					return false, nil
+				}
+				v, ok := tac.Annotations[label.AnnTiKVLastAutoScalingTimestamp]
+				if !ok {
+					klog.Infof("tac haven't marked any annotation")
+					return false, nil
+				}
+				firstScaleTimestamp, err = strconv.ParseInt(v, 10, 64)
+				if err != nil {
+					return false, err
+				}
+				// check phase back to norma
+				framework.ExpectEqual(tac.Status.TiKV.Phase, v1alpha1.NormalAutoScalerPhase)
+				// check store label
+				storeId := ""
+				for k, v := range tc.Status.TiKV.Stores {
+					if v.PodName == util.GetPodName(tc, v1alpha1.TiKVMemberType, int32(3)) {
+						storeId = k
+						break
+					}
+				}
+				if storeId == "" {
+					return false, nil
+				}
+				sid, err := strconv.ParseUint(storeId, 10, 64)
+				if err != nil {
+					return false, err
+				}
+				info, err := pdClient.GetStore(sid)
+				if err != nil {
+					return false, nil
+				}
+				for _, label := range info.Store.Labels {
+					if label.Key == "specialUse" && label.Value == "hotRegion" {
+						return true, nil
+					}
+				}
+				klog.Infof("tikv auto-scale out haven't find the special label")
+				return false, nil
```

Comment:
What about merge noise reduction test case into the existed tikv auto-scaling e2e test ? We don't have to create a new e2e test case.